src/sentry/templates/sentry/organization-login.html

```diff
@@ -4,6 +4,93 @@
 {% load i18n %}
 {% load sentry_assets %}
 
+{% block scripts %}
+{{ block.super }}
+
+<script>
+  (function() {
+    var ZXCVBN_SRC = "{% asset_url 'sentry' 'js/zxcvbn.js' %}";
+
+    var async_load = function() {
+      var first, s;
+      s = document.createElement('script');
+      s.src = ZXCVBN_SRC;
+      s.type = 'text/javascript';
+      s.async = true;
+      first = document.getElementsByTagName('script')[0];
+      return first.parentNode.insertBefore(s, first);
+    };
+
+    if (window.attachEvent != null) {
+      window.attachEvent('onload', async_load);
+    } else {
+      window.addEventListener('load', async_load, false);
+    }
+  }).call(this);
```

Comment:
Not used to seeing IIFEs with `.call` ... is there a benefit to this over just doing `})();`?